vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -1112,7 +1090,8 @@ def select_gemm_impl(
         layer: torch.nn.Module,
```

Comment:
If `BatchedTritonOrDeepGemmExperts` is not called elsewhere, could we delete the whole class?